Scheduled functional test failed - Run ID: 6581917730

## Bug information 

This bug is generated automatically if the scheduled functional test fails. The Radius functional test operates on a schedule of every 4 hours during weekdays and every 12 hours over the weekend. It's important to understand that the test may fail due to workflow infrastructure issues, like network problems, rather than the flakiness of the test itself. For the further investigation, please visit [here](https://github.com/radius-project/radius/actions/runs/6581917730).

[AB#9872](https://dev.azure.com/azure-octo/e61041b4-555f-47ae-95b2-4f8ab480ea57/_workitems/edit/9872)

```
=== Failed
=== FAIL: test/functional/shared/mechanics Test_AWSRedeployWithUpdatedResourceUpdatesResource/deploy_testdata/aws-mechanics-redeploy-withupdatedresource.step1.bicep (49.81s)
    rptest.go:272: running step 0 of 2: deploy testdata/aws-mechanics-redeploy-withupdatedresource.step1.bicep
    deployexecutor.go:80: deploying deploy testdata/aws-mechanics-redeploy-withupdatedresource.step1.bicep from file testdata/aws-mechanics-redeploy-withupdatedresource.step1.bicep
    cli.go:468: [heartbeat] command rad deploy /home/runner/work/radius/radius/test/functional/shared/mechanics/testdata/aws-mechanics-redeploy-withupdatedresource.step1.bicep --parameters bucketName=radiusfunctionaltestbucket-628289a7-3049-4c33-a202-33c7af9d8fa9 --parameters creationTimestamp=1697763575 is still running after 10.004507226s
    cli.go:468: [heartbeat] command rad deploy /home/runner/work/radius/radius/test/functional/shared/mechanics/testdata/aws-mechanics-redeploy-withupdatedresource.step1.bicep --parameters bucketName=radiusfunctionaltestbucket-628289a7-3049-4c33-a202-33c7af9d8fa9 --parameters creationTimestamp=1697763575 is still running after 20.004795725s
    cli.go:468: [heartbeat] command rad deploy /home/runner/work/radius/radius/test/functional/shared/mechanics/testdata/aws-mechanics-redeploy-withupdatedresource.step1.bicep --parameters bucketName=radiusfunctionaltestbucket-628289a7-3049-4c33-a202-33c7af9d8fa9 --parameters creationTimestamp=1697763575 is still running after 30.005967165s
    cli.go:468: [heartbeat] command rad deploy /home/runner/work/radius/radius/test/functional/shared/mechanics/testdata/aws-mechanics-redeploy-withupdatedresource.step1.bicep --parameters bucketName=radiusfunctionaltestbucket-628289a7-3049-4c33-a202-33c7af9d8fa9 --parameters creationTimestamp=1697763575 is still running after 40.006496613s
2023/10/20 01:00:15 Start streaming Kubernetes logs - Pod cyclea-68dd88d5d9-587f2 is in state: Failed
2023/10/20 01:00:16 Start streaming Kubernetes logs - Pod mechanicsg-6b8ddb879c-l6kgt is in state: Running
2023/10/20 01:00:16 Saved container logs to /home/runner/work/radius/radius/dist/container_logs/cyclea-68dd88d5d9-587f2.cyclea.log
2023/10/20 01:00:16 Start streaming Kubernetes logs - Pod cyclea-68dd88d5d9-587f2 is in state: Failed
2023/10/20 01:00:16 Start streaming Kubernetes logs - Pod cyclea-68dd88d5d9-587f2 is in state: Failed
2023/10/20 01:00:16 Start streaming Kubernetes logs - Pod cyclea-68dd88d5d9-587f2 is in state: Failed
2023/10/20 01:00:16 Start streaming Kubernetes logs - Pod mechanicsg-6b8ddb879c-l6kgt is in state: Failed
2023/10/20 01:00:17 Saved container logs to /home/runner/work/radius/radius/dist/container_logs/mechanicsg-6b8ddb879c-l6kgt.mechanicsg.log
2023/10/20 01:00:17 Start streaming Kubernetes logs - Pod mechanicsg-6b8ddb879c-l6kgt is in state: Failed
2023/10/20 01:00:17 Start streaming Kubernetes logs - Pod mechanicsg-6b8ddb879c-l6kgt is in state: Failed
2023/10/20 01:00:17 Start streaming Kubernetes logs - Pod mechanicsg-6b8ddb879c-l6kgt is in state: Failed
    cli.go:415: [rad] Building /home/runner/work/radius/radius/test/functional/shared/mechanics/testdata/aws-mechanics-redeploy-withupdatedresource.step1.bicep...
    cli.go:415: [rad] Deploying template '/home/runner/work/radius/radius/test/functional/shared/mechanics/testdata/aws-mechanics-redeploy-withupdatedresource.step1.bicep' into environment 'kind-radius' from workspace 'kind-radius'...
    cli.go:415: [rad] 
    cli.go:415: [rad] Deployment In Progress...
    cli.go:415: [rad] 
    cli.go:415: [rad] Error: {
    cli.go:415: [rad]   "code": "DeploymentFailed",
    cli.go:415: [rad]   "message": "At least one resource deployment operation failed. Please see the details for the specific operation that failed.",
    cli.go:415: [rad]   "target": "/planes/radius/local/resourceGroups/kind-radius/providers/Microsoft.Resources/deployments/rad-deploy-d61a7309-556b-4f79-b17a-95775d00c34b",
    cli.go:415: [rad]   "details": [
    cli.go:415: [rad]     {
    cli.go:415: [rad]       "code": "BadRequest",
    cli.go:415: [rad]       "message": "mandatory property Id is missing",
    cli.go:415: [rad]       "target": "/planes/aws/aws/accounts/***/regions/us-west-2/providers/AWS.S3/Bucket/radiusfunctionaltestbucket-628289a7-3049-4c33-a202-33c7af9d8fa9"
    cli.go:415: [rad]     }
    cli.go:415: [rad]   ]
    cli.go:415: [rad] }
    cli.go:415: [rad] 
    cli.go:415: [rad] TraceId:  967742c947df32d79859d2175fd523f8
    cli.go:415: [rad] 
    cli.go:415: [rad] 
    deployexecutor.go:83: 
        	Error Trace:	/home/runner/work/radius/radius/test/step/deployexecutor.go:83
        	            				/home/runner/work/radius/radius/test/functional/shared/rptest.go:273
        	Error:      	Received unexpected error:
        	            	code DeploymentFailed: err At least one resource deployment operation failed. Please see the details for the specific operation that failed.
        	Test:       	Test_AWSRedeployWithUpdatedResourceUpdatesResource/deploy_testdata/aws-mechanics-redeploy-withupdatedresource.step1.bicep
        	Messages:   	failed to deploy deploy testdata/aws-mechanics-redeploy-withupdatedresource.step1.bicep
    --- FAIL: Test_AWSRedeployWithUpdatedResourceUpdatesResource/deploy_testdata/aws-mechanics-redeploy-withupdatedresource.step1.bicep (49.81s)

```

--- a/test/functional/shared/mechanics/aws_mechanics_test.go
+++ b/test/functional/shared/mechanics/aws_mechanics_test.go
@@ -28,6 +28,7 @@ import (
 )
 
 func Test_AWSRedeployWithUpdatedResourceUpdatesResource(t *testing.T) {
+	t.Skip("Skipping until we resolve https://github.com/radius-project/radius/issues/6535")
 	templateFmt := "testdata/aws-mechanics-redeploy-withupdatedresource.step%d.bicep"
 	name := "radiusfunctionaltestbucket-" + uuid.New().String()
 	creationTimestamp := functional.GetCreationTimestamp()

--- a/test/functional/shared/resources/aws_s3_bucket_test.go
+++ b/test/functional/shared/resources/aws_s3_bucket_test.go
@@ -26,6 +26,7 @@ import (
 )
 
 func Test_AWS_S3Bucket(t *testing.T) {
+	t.Skip("Skipping until we resolve https://github.com/radius-project/radius/issues/6535")
 	template := "testdata/aws-s3-bucket.bicep"
 	name := functional.GenerateS3BucketName()
 	creationTimestamp := functional.GetCreationTimestamp()
@@ -64,6 +65,7 @@ func Test_AWS_S3Bucket(t *testing.T) {
 }
 
 func Test_AWS_S3Bucket_Existing(t *testing.T) {
+	t.Skip("Skipping until we resolve https://github.com/radius-project/radius/issues/6535")
 	template := "testdata/aws-s3-bucket.bicep"
 	templateExisting := "testdata/aws-s3-bucket-existing.bicep"
 	name := functional.GenerateS3BucketName()

--- a/test/functional/shared/resources/extender_test.go
+++ b/test/functional/shared/resources/extender_test.go
@@ -98,6 +98,7 @@ func Test_Extender_Recipe(t *testing.T) {
 }
 
 func Test_Extender_RecipeAWS(t *testing.T) {
+	t.Skip("Skipping until we resolve https://github.com/radius-project/radius/issues/6535")
 	awsAccountID := os.Getenv("AWS_ACCOUNT_ID")
 	awsRegion := os.Getenv("AWS_REGION")
 	// Error the test if the required environment variables are not set

--- a/test/functional/ucp/aws_test.go
+++ b/test/functional/ucp/aws_test.go
@@ -44,7 +44,7 @@ var (
 )
 
 func Test_AWS_DeleteResource(t *testing.T) {
-	t.Skip("https://github.com/radius-project/radius/issues/5967")
+	t.Skip("Skipping until we resolve https://github.com/radius-project/radius/issues/6535")
 	ctx := context.Background()
 
 	bucketName := generateS3BucketName()
@@ -106,7 +106,7 @@ func Test_AWS_DeleteResource(t *testing.T) {
 }
 
 func Test_AWS_ListResources(t *testing.T) {
-	t.Skip("https://github.com/radius-project/radius/issues/5967")
+	t.Skip("Skipping until we resolve https://github.com/radius-project/radius/issues/6535")
 	ctx := context.Background()
 
 	var bucketName = generateS3BucketName()

